developersociety/django-template
# add custom User model

# Sources
Cluster meeting

# Description
Adds custom user model 

# Setup instructions
N/A, no need to clone this one.

# How to test
1. This is a scaffold, so just check the files changed, make sure they follow our standard procedure for adding custom user models.

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-f804361f9b6b4859bdcd69d7704168e5.

# Pre-review checklist
